### PR TITLE
feature/additions_and_new_streets_ios

### PR DESCRIPTION
**Pull request by xidealo**

### CODE REVIEW

File: iosApp/iosApp/custom_view/iOSCheckboxToggleStyle.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    func makeBody(configuration: Configuration) -> some View {
+        // 1
+        Button(action: {
+
+            // 2
+            configuration.isOn.toggle()
+
+        }, label: {
+            HStack {
+                // 3
```
Comment (MaxAstin) [inline iosApp/iosApp/custom_view/iOSCheckboxToggleStyle.swift:22]: Let's remove extra comments //1 //2 //3

File: iosApp/iosApp/feature/address/create_address/CreateAddressView.swift [hunk 2]
```diff
@@ -8,104 +8,100 @@
...
-    @State var showError:Bool = false
     
-    @StateObject var viewModel:CreateAddressHolder = CreateAddressHolder()
+    @State var viewModel = CreateAddressViewModel(
+        getSuggestionsUseCase: iosComponent.provideGetSuggestionsUseCase(),
+        createAddressUseCase: iosComponent.provideCreateAddressUseCase(),
+        saveSelectedUserAddressUseCase: iosComponent.provideSaveSelectedUserAddressUseCase()
+    )
     
+    //Listeners
```
Comment (MaxAstin) [inline iosApp/iosApp/feature/address/create_address/CreateAddressView.swift:28]: Let's remove extra comments

File: iosApp/iosApp/feature/auth/login/LoginView.swift [hunk 3]
```diff
@@ -30,6 +30,7 @@ struct LoginView: View {
     @State var hasPhoneError: Bool = false
     @State var phone = ""
     @State var phoneNumberCursorPosition = Int32(shared.Constants().PHONE_CODE.count)
+    @State var textFieldError : LocalizedStringKey? = nil
     // ---
```
Comment (MaxAstin) [inline iosApp/iosApp/feature/auth/login/LoginView.swift:34]: Let's remove extra comments

File: iosApp/iosApp/feature/consumer_cart/CartProductView.swift [hunk 4]
```diff
@@ -11,49 +11,89 @@ import shared
...
+    @State var openProductDetails:Bool = false
+    @Binding var created:Bool
+    @Binding var edited:Bool
+    
+    // ----
+    
+    // Actions
     let plusAction: () -> Void
     let minusAction: () -> Void
+    // ----
```
Comment (MaxAstin) [inline iosApp/iosApp/feature/consumer_cart/CartProductView.swift:29]: Let's remove extra comments

File: iosApp/iosApp/feature/consumer_cart/ConsumerCartView.swift [hunk 5]
```diff
@@ -39,6 +40,11 @@ struct ConsumerCartView: View {
     @Binding var showOrderCreated:Bool
     //--
     
+    //for add or edit product
+    @State var created:Bool = false
+    @State var edited:Bool = false
+    //--
```
Comment (MaxAstin) [inline iosApp/iosApp/feature/consumer_cart/ConsumerCartView.swift:46]: Let's remove extra comments

File: shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/consumercart/ConsumerCartViewModel.kt [hunk 6]
```diff
@@ -254,9 +259,9 @@ class ConsumerCartViewModel(
                     )
                 },
                 oldTotalCost = consumerCartDomain.oldTotalCost?.let { oldTotalCost ->
-                    "$oldTotalCost $RUBLE_CURRENCY"
+                    "$oldTotalCost$RUBLE_CURRENCY"
                 },
-                newTotalCost = "${consumerCartDomain.newTotalCost} $RUBLE_CURRENCY",
+                newTotalCost = "${consumerCartDomain.newTotalCost}$RUBLE_CURRENCY",
```
Comment (MaxAstin) [inline shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/consumercart/ConsumerCartViewModel.kt:264]: Let's leave a space here and update the constant as `RUBLE_CURRENCY = "₽"`

**Review by MaxAstin — CHANGES_REQUESTED** [review]
Pls, fix comments

File: iosApp/iosApp/custom_view/iOSCheckboxToggleStyle.swift (hunk 1 above)
Comment (xidealo) [reply iosApp/iosApp/custom_view/iOSCheckboxToggleStyle.swift:22]: removed this class

File: iosApp/iosApp/feature/address/create_address/CreateAddressView.swift (hunk 2 above)
Comment (xidealo) [reply iosApp/iosApp/feature/address/create_address/CreateAddressView.swift:28]: this is codestyle for me

**Pull request merged**